src/test/java/integration/com/vendingontime/backend/repositories/JPAMachineRepositoryTest.java

```diff
@@ -196,4 +197,15 @@ public void findByCompany_companyExistsWithMachines_returnsMachines() throws Exc
         assertThat(machines.contains(savedMachine1), is(true));
         assertThat(machines.contains(savedMachine2), is(true));
     }
+
+    @Test
+    public void deleteAll() throws Exception {
+        Machine machine = repository.create(this.machineOne);
+        String machineId = machine.getId();
+
+        repository.deleteAll();
+
+        Optional<Machine> possiblePerson = repository.findById(machineId);
+        assertFalse(possiblePerson.isPresent());
```

Comment:
Cada vez me usas una cosa. A veces usas assertThat + is y otras assertFalse/assertTrue. Intentemos mantener un convenio.